app/frontend/src/submissions/components/SubmissionForm/WellType.vue

```diff
@@ -110,14 +110,18 @@ Licensed under the Apache License, Version 2.0 (the "License");
       </b-row>
       <b-row>
         <b-col cols="12" md="4"  v-if="!isStaffEdit">
-          <b-form-group label="Well Tag Number (if known)">
+          <b-form-group>
+            <label>Well Tag Number (if known) <span class="font-weight-bold">*displays a maximum of {{MAX_RESULTS}} results</span></label>
```

Comment:
you could alternatively add this as a form hint
```:hint="`Displays a maximum of ${MAX_RESULTS}`"```
to give it a cleaner look... right now it kind of dominates the actual label. Maybe Lindsay/Nicole can decide.